client/my-sites/site-settings/disconnect-site/another-reason.jsx

```diff
@@ -0,0 +1,55 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormTextArea from 'components/forms/form-textarea';
+import SectionHeader from 'components/section-header';
+import { addQueryArgs } from 'lib/url';
+
+class AnotherReason extends PureComponent {
+	state = {
+		reason: '',
+	};
+
+	handleChange = event => {
+		this.setState( { reason: event.target.value } );
+	};
+
+	render() {
+		const { confirmHref, translate } = this.props;
+
+		return (
+			<div className="disconnect-site__another-reason">
+				<SectionHeader
+					label={ translate(
+						'Please share why you want to disconnect your site from WordPress.com.'
+					) }
+				/>
+				<Card>
+					<FormTextArea onChange={ this.handleChange } value={ this.state.reason } />
```

Comment:
Also, it seems the textarea can be resized horizontally, which might bring some unexpected results:

![](https://cldup.com/HZlwl0ozLa.png)

Perhaps we want to limit it to be resized only vertically.